txscript/stdaddr/addressv0.go

```diff
@@ -1106,14 +1106,36 @@ func (addr *AddressScriptHashV0) Hash160() *[ripemd160.Size]byte {
 // relevant Address if it is a valid encoding for a known version 0 address type
 // and is for the network identified by the provided parameters.
 func DecodeAddressV0(addr string, params AddressParamsV0) (Address, error) {
+	// The provided address must not be larger than the maximum possible size.
+	//
+	// The largest supported version 0 decoded address data consists of 33 bytes
+	// for the public key, 2 bytes for the network identifier, and 4 bytes for
+	// the checksum.
+	//
+	// Since the encoding converts from base256 to base58, the max possible
+	// number of bytes of output per input byte is log_58(256) ~= 1.37.  Thus, a
+	// reasonable estimate for the max possible encoded size is
+	// ceil(decodedDataLen * 1.37).
```

Comment:
You reference `decodedDataLen` here, but you named the associated const `maxV0AddrLen` for this case.

Reply:
Good catch.  Updated.

Reply:
Actually, I had just defined the final constant here instead of the decoded data length, because it's not the same size for all address types.  `maxV0AddrLen` is the final value.

I went ahead and changed the comment to read `ceil(max decoded data len * 1.37)`. to hopefully make it more clear.

Reply:
Yeah, that makes sense.   The latest update looks good to me.